javascript/src/main/pages/Statistics/DivisionOccupancy.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { useState } from "react";
+import { Jumbotron } from "react-bootstrap";
+import { fetchDivisionOccupancy } from "main/services/statisticsService";
+import DivisionOccupancyForm from "main/components/Statistics/DivisionOccupancyForm";
+import DivisionOccupancyTable from "main/components/Statistics/DivisionOccupancyTable";
+import { fromFormat } from "main/components/Statistics/QuarterFormSelect";
+
+const DivisionOccupancy = () => {
+    const [courseJSON, setCourseJSON] = useState([]);
+    const [formSubmitted, setFormSubmitted] = useState(false);
+    const setInitialData = (courseJSON) => {
+        let newArray = courseJSON.map((item, index) => ({index, ...item}));
```

Comment:
Unclear functionality of these transformations on the json data. Consider adding comments, or into 
a separate function with explanation.